### PR TITLE
Don't allow description track to satisfy media SCs

### PR DESCRIPTION
**Pull request by WilcoFiers**
<< Describe the changes >>

Closes issue(s):

- closes #XXX (ADD ISSUE NUMBER HERE)

Need for call for review: 1 week

---

### **When creating PR:**

- [ ] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [ ] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [ ] Add yourself (and co-authors) as "Assignees" for PR.
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [ ] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by colabottles — APPROVED** [review]

File: _rules/video-strict-alternative-for-visual-1ec09b.md [hunk 1]
```diff
@@ -133,29 +110,29 @@ A video element with an incorrect audio description.
 
 #### Failed Example 2
 
-A video element with a track element that contains incorrect descriptions.
+A video element with a link to a text transcript.
```
Comment (ajanec01) [inline _rules/video-strict-alternative-for-visual-1ec09b.md:113]: I'm not sure how is that failing the rule. Where is the requirement that a text transcript needs to be on the same page?

**Comment by kengdoj** [comment]
@WilcoFiers notified AG when a technique was not accessibility supported ([H69: headings](https://www.w3.org/WAI/WCAG21/Techniques/html/H69.html) in Issue #1587). Do we need to do the same with [H95](https://www.w3.org/WAI/WCAG21/Techniques/html/H95.html) before we remove it from ACT rules?

**Comment by kengdoj** [comment]
https://github.com/w3c/wcag/issues/599 discussed H69 not working as expected and is closed. Below was added to technique:

> As of February 2019 when this Advisory technique was last reviewed by the Working Group, there is no native support in user agents for this technique. However, support is available via JavaScript polyfills.

File: _rules/video-strict-alternative-for-visual-1ec09b.md (hunk 1 above)
Comment (kengdoj) [reply _rules/video-strict-alternative-for-visual-1ec09b.md:113]: I think this example is correct. A transcript wouldn't be an acceptable alternative for video with audio.

**Review by colabottles — APPROVED** [review]

File: _rules/video-strict-alternative-for-visual-1ec09b.md (hunk 1 above)
Comment (ajanec01) [reply _rules/video-strict-alternative-for-visual-1ec09b.md:113]: Sorry, got that wrong.

**Review by ajanec01 — APPROVED** [review]

**Review by daniel-montalvo — APPROVED** [review]

**Pull request merged**